mmv1/third_party/terraform/services/compute/resource_compute_region_instance_group_manager_test.go.erb

```diff
@@ -414,25 +414,6 @@ func TestAccRegionInstanceGroupManager_stoppedSuspendedTargetSize(t *testing.T)
 }
 <% end -%>
 
-func TestAccRegionInstanceGroupManager_APISideListRecordering(t *testing.T) {
```

Comment:
I wonder if we want to keep this test to ensure the change from list to set works, like it also wouldn't cause permadiff for multiple stateful_internal_ip/stateful_external_ip attributes configurations.